scm-ui/ui-webapp/src/repos/containers/RepositoryRoot.tsx

```diff
@@ -120,7 +119,7 @@ const RepositoryRoot = () => {
   if (redirectUrlFactory) {
     redirectedUrl = url + redirectUrlFactory(props);
   } else {
-    redirectedUrl = url + "/info";
+    redirectedUrl = url + "/code/sources/";
```

Comment:
Linking directly to the source code makes sense most of the time. But it does not make sense for the redirect after the repository was created. I see two options for the prolem:

1. Redirect to the information after repository creation
2. Add checkout/clone information to the code page, if the repository is empty

Reply:
i prefer the first solution